processor/elasticinframetricsprocessor/README.md

```diff
@@ -0,0 +1,34 @@
+# Elastic Infra Metrics Processor
+
+<!-- status autogenerated section -->
+| Status        |           |
+| ------------- |-----------|
+| Stability     | [alpha]:  metrics  |
+| Distributions |  |
+| [Code Owners](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/CONTRIBUTING.md#becoming-a-code-owner)    | [@ishleenk17](https://www.github.com/ishleenk17) |
+
+[beta]: https://github.com/open-telemetry/opentelemetry-collector#beta
+[core]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol
+[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+<!-- end autogenerated section -->
+
+The Elastic Infra Metrics Processor can be used to bridge the gap between OTEL and Elastic Infra Metrics. It is used to power the Curated UI's in Elastic. 
+This processor uses the [elastic/opentelemtery-lib](https://github.com/elastic/opentelemetry-lib) library, which derives and adds Elastic compatible metrics from the OTEL metrics without altering the OTEL metrics. 
+The processor should be used only with the below receivers and exporter.
+
+**Receivers**: It should be used with the [Hostmetrics receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/README.md), [Kubeletstatsreceiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver).<br>
```

Comment:
```suggestion
**Receivers**: It should be used with the [Hostmetrics receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/README.md) and [Kubelet Stats Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kubeletstatsreceiver#kubelet-stats-receiver).<br>
```

Reply:
That's still valid: https://github.com/elastic/opentelemetry-collector-components/pull/15#discussion_r1646338850